src/lib.rs

```diff
@@ -493,6 +493,10 @@ use autocxx_engine::IncludeCppEngine;
 /// }
 /// ```
 ///
+/// Also note that if you place the `cxx::bridge` in a separate module (such
+/// as for tests), you will need a [`include_cpp`] in that module in order for
```

Comment:
I'm working on support for other modules right now. I'm not sure if this is going to be true or not.

Thinking about it, I'll remove this from this PR, and put it in the PR that actually supports `include_cpp!` in other modules instead.